fendermint/testing/contract-test/tests/run_upgrades.rs

```diff
@@ -235,7 +237,7 @@ async fn test_applying_upgrades() {
         tester.commit().await.unwrap();
 
         // check that the app_version was upgraded to 1
-        assert_eq!(tester.state_params().app_version, 1);
+        assert_eq!(tester.state_params().app_version, block_height as u64 + 10);
```

Comment:
I think `block_height + 10` as `app_version` is puzzling. I kept wondering why that should be true, and it's true in this particular set of hardcoded values, but it looks like some kind of rule that one has to make sense of. Perhaps it would be more straight forward to just iterate over the known list of (height, version) pairs and assert them, rather than have a formula for it that works now, but it's not clear where the upgrades are defined that this offset _must_ be adhered to.